README.md

```diff
@@ -50,6 +50,7 @@ See some examples [in the Explainer file](explainer.md).
 
 ## <a name='History'></a>History
 
+* we moved on from the exposed globalThis model to a lean isolated realms API (see #289 and #291)
```

Comment:
isolated realms => Realm with Callable Boundary or something like that...

we chatted about this with Shu, and I believe we should avoid using the term isolated realm all together.

Reply:
Another alternative is also "Callable Realms"

Reply:
I find the names that @caridy is suggesting confusing, though avoiding the word "isolated" could be useful to avoid other kinds of confusion. Maybe we could avoid giving this version a name for now, until we're really confident in one, since it's awkward to change names later.